modules/private-clusters-default.adoc

```diff
@@ -18,15 +18,19 @@ ifeval::["{context}" == "installing-aws-china-region"]
 :aws-china:
 endif::[]
 
+ifeval::["{context}" == "installing-aws-secret-region"]
+:aws-secret:
+endif::[]
+
 You can deploy a private {product-title} cluster that does not expose external endpoints. Private clusters are accessible from only an internal network and are not visible to the internet.
 
-ifdef::aws-gov[]
+ifdef::aws-gov,aws-secret[]
 [NOTE]
 ====
 Public zones are not supported in Route 53 in AWS GovCloud or Secret Regions. Therefore, clusters
```

Comment:
Conditional logic needed here to mention just GovCloud or Secret, not both.

Reply:
Agreed and done.